src/components/generic/OptionsList/OptionsList.scss

```diff
@@ -1,18 +1,7 @@
 $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-options-list;
 
 .#{$C} {
-  border-radius: 0.75rem;
-  box-shadow: var(--wxc-options-list--normal--box-shadow);
-  background: var(--wxc-options-list--normal--background);
-  color: var(--wxc-options-list--normal--color);
-  border: var(--wxc-options-list--normal--border);
-  padding: 0.5rem;
-  overflow: auto;
-  min-height: 1.5rem;
-
-  .#{$C}__list {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-  }
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
can we keep the border, box-shadow etc on this component? maybe other popups look different

Reply:
actually, we can't, because of the scrollbar that must belong to the element having the box shadow and borders